docs/notebooks/feasible_sets.pct.py

```diff
@@ -230,6 +237,58 @@ def objective_function(x):
 )
 
 
+# %% [markdown]
+# ## Active learning with Integrated Variance Reduction acquisition function
+#
+# An alternative to the `ExpectedFeasibility` function is called `IntegratedVarianceReduction`. This acquisition has the advantage of allowing batches, as we illustrate below. It is more expensive to compute than `ExpectedFeasibility`, as it writes as an integral over a set of input locations.
+
+# %%
+from trieste.acquisition.function import IntegratedVarianceReduction
+
+# Choose integration points uniformly over the design space
+integration_points = search_space.sample_halton(1000)
+acq_ivr = IntegratedVarianceReduction(integration_points=integration_points, threshold=[threshold,])
+
+# Set a batch size greater than 1
+rule_ivr = EfficientGlobalOptimization(builder=acq_ivr, num_query_points=5) # type: ignore
```

Comment:
I don't know why I needed to add `type: ignore`, but otherwise `tox` complains: "error: Need type annotation for "rule_ivr""

and I have no idea why